src/tracked.jl

```diff
@@ -270,10 +270,22 @@ Base.convert(::Type{T}, t::T) where {T<:TrackedReal} = t
 Base.convert(::Type{T}, t::T) where {T<:TrackedArray} = t
 
 for R in REAL_TYPES
-    @eval Base.promote_rule(::Type{$R}, ::Type{TrackedReal{V,D,O}}) where {V,D,O} = TrackedReal{promote_type($R,V),D,O}
+    R === :Dual && continue # ForwardDiff.Dual is handled below 
```

Comment:
Only defining `promote_rule` for `Type{<:Dual}` leads to method ambiguity issues; hence it is skipped here and defined below for `Type{Dual{T,V,N}}`.